script/s3.js

```diff
@@ -0,0 +1,95 @@
+const S3 = require("aws-sdk/clients/s3");
+const Cloudfront = require("aws-sdk/clients/cloudfront");
+const fs = require("fs");
+const awsCredentials = require("./aws.js");
+const path = require("path");
```

Comment:
```js
const S3 = require("aws-sdk/clients/s3"),
         Cloudfront = require("aws-sdk/clients/cloudfront"),
         fs = require("fs"),
         awsCredentials = require("./aws.js"),
         path = require("path");
```

Reply:
hmmmm  I don't really like this approach. Although it reduces repetition of the word `const` I think it also reduces readability

Reply:
what do you guys think?

Reply:
Since it's a Node script, that's their standard when importing modules